### PR TITLE
Add draft authors

### PR DESCRIPTION
**Pull request by Robinlovelace**
Please check and review @anitagraser, @michaeldorman, @Nowosad

### CODE REVIEW

File: blog.qmd [hunk 1]
```diff
@@ -3,6 +3,23 @@ title: "Geographic data analysis in R and Python: comparing code and outputs for
...
+  - name: Anita Graser
+    affiliation: Austrian Institute of Technology, Austria
+    orcid: 0000-0001-5361-2885
+    email: 
+  - name: Michael Dorman
+    affiliation: Ben-Gurion University of the Negev, Israel
+    orcid: 
+    email:
+  - name: Jakub Nowosad
+    affiliation: Adam Mickiewicz University in Poznań, Poland
```
Comment (Nowosad) [inline blog.qmd:20]:
```suggestion
    affiliation: Adam Mickiewicz University, Poznań, Poland
```

File: blog.qmd [hunk 2]
```diff
@@ -60,6 +77,10 @@ import geopandas as gpd
 
 <!---
  jn: I would suggest to use specific packages from tidyverse instead of attaching the whole tidyverse 
+
+rl: Why? Tidyverse is popular and it makes life easy.
+Also that's the approach in the teaching materials.
+For the book and for software development that's another matter but for this blog post I think it's fine.
```
Comment (Nowosad) [inline blog.qmd:83]: Sure -- I usually would not do that, but it is fine. Just a suggesion

File: blog.qmd [hunk 3]
```diff
@@ -517,11 +517,36 @@ buffers = gpd.GeoDataFrame({
 buffers.sjoin(points2, how='left')
 ```
 
+# Questions and next steps
 
+The code above shows that R and Python are similar in many ways.
+Differences include:
+
+- R's use of the S2 spherical geometry engine by default for lon/lat data, which can mean fewer lines of code for some operations (e.g. buffers)
+- R returns measures with units, which can make conversions easier, but which can also be confusing for new users
```
Comment (Nowosad) [inline blog.qmd:526]: Same as above

File: blog.qmd [hunk 4]
```diff
@@ -517,11 +517,36 @@ buffers = gpd.GeoDataFrame({
...
+It also raises some questions, which we leave unanswered for the community to consider and comment on:
+
+- Which language is more concise?
+  - While there are slightly fewer lines of R code in the examples above, there may be ways to improve the Python code
+- Which language runs quicker?
+    - It would be interesting to see benchmarks for the above code, perhaps in a future [geocompx](https://geocompx.org/) blog post or even a multi-language benchmarking package
+- Which language is quicker to write code in (the answer likely depends on your prior experience and tastes)?
+
+We welcome input on these questions and any other comments on the above code, the source code of which can be found at [github.com/geocompx/geocompx.org](https://github.com/geocompx/geocompx.org/blob/main/post/2023/rgdal-retirement/index.qmd).
+If you would like to contribute, with ideas, comments, or additional questions, feel free to get in touch via the [geocompx](https://geocompx.org/) website, our [Discord server](https://discord.gg/PMztXYgNxp), in a [GitHub Discussion](https://github.com/geocompx/geocompx.org/discussions), on [Mastodon](https://fosstodon.org/tags/geocompx) or anywhere else.
```
Comment (Nowosad) [inline blog.qmd:542]: https://github.com/orgs/geocompx/discussions

I think the above link is better (easier to remember) -- it shows the same discussion

Comment (Robinlovelace) [reply blog.qmd:540]: Agreed. Will add.

File: blog.qmd [hunk 5]
```diff
@@ -400,8 +400,7 @@ stops_projected = st_transform(stops, 2180)
 
 The following commands create buffers around the points.
 Note that R allows buffer to be created directly from a spatial data frame with geographic (lon/lot) coordinates thanks to its integration with Google's S2 spherical geometry engine, as outlined in [Geocomputation with R](https://r.geocompx.org/reproj-geo-data#geom-proj).
-For buffer operations to work in Python you must reproject the data first (although there plans for GeoPandas to support a spherical geometry backend at some point, as discussed in issue [#2098](https://github.com/geopandas/geopandas/issues/2098)).
-
+For buffer operations to work in Python you must reproject the data first (which we did, see above) (although there plans for **geopandas** to support a spherical geometry backend at some point, as discussed in issue [#2098](https://github.com/geopandas/geopandas/issues/2098)).
```
Comment (Nowosad) [inline blog.qmd:403]:
```suggestion
For buffer operations to work in Python you must reproject the data first (which we did, see above) (although there are plans for **geopandas** to support a spherical geometry backend at some point, as discussed in issue [#2098](https://github.com/geopandas/geopandas/issues/2098)).
```

**Comment by Robinlovelace** [comment]
Great review Jakub. Going to merge so CI can do its thing over lunch.

**Comment by Nowosad** [comment]
Awesome.

**Pull request merged**